src/DataModelRegistry.cpp

```diff
@@ -54,7 +54,7 @@ getTypeConverter(QString const &sourceTypeID, QString const &destTypeID) const
 
   if (converter != _registeredTypeConverters.end())
   {
-    return converter->second->Model->clone();
+    return converter->second->ModelCloner();
```

Comment:
I recommend using the name `ModelCreator` rather than `ModelCloner`. We are removing the `clone()` operation in favor of a factory function (there's `_registeredModelCreators`)

This is a member of `TypeConverterItem`, defined [here](https://github.com/paceholder/nodeeditor/pull/135/files#diff-3836c6057311bf852253b1fb88143a55R31)